### ISSUE TITLE
About the training loss

### ISSUE THREAD

**Issue by huinsysu** [zhaoweicai/cascade-rcnn#54]
Hi, I have trained the resnet50+fpn+cascade model with my own datasets. In my datasets, there are 189 classes and about 12000 samples. I keep the solver.prototxt same as yours and make some small change to the train.prototext. But when I finish the training, my training loss is about 0.15, which I think is too large. And I ran the detection code on some picture and found that the scores for the bbox were too low. I want to ask you what the normal training loss should be and Is it the problem of my learning rate resulting such bad performance?(should I lower it?) Thanks!

**Comment by Shadow992** [comment]
Just for future people stumbling over this, have also a look at:

https://github.com/zhaoweicai/cascade-rcnn/issues/49

We discussed there a bit, too.
